services/error.go

```diff
@@ -17,6 +20,10 @@ var (
 	ErrObjectNotExist = errors.New("object not exist")
 	// ErrPermissionDenied means this operation doesn't have enough permission.
 	ErrPermissionDenied = errors.New("permission denied")
+	// ErrObjectNotPart means the object to be operated is not a part object.
+	ErrObjectNotPart = errors.New("object is not a part object")
```

Comment:
We do have `not part`, `not append`, `not page`, `not read`, they are all `invalid object mode`.

Maybe we can use the same error code here?